apps/whale/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3.7'
```

Comment:
Is this compose for doing test in local?

It is a mixed of production (port) and test (regtest flag) configuration, not sure what's the purpose.

Reply:
Woopsies - meant to update that
Thanks for the catch

Reply:
yes this is currently purely for local sanity testing https://github.com/DeFiCh/whale/blob/main/.github/workflows/ci/docker-compose.yml

This will most likely be re-worked in upcoming PRs in regards to sanity testing introductions.

Reply:
And yes, it's meant for local run only like rich-list-api's